jupyter/trustyai/ubi9-python-3.9/test/test_notebooks.ipynb

```diff
@@ -0,0 +1,133 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "df03ef8f-cfd8-42d6-8d37-28c92ecb63b0",
+   "metadata": {},
+   "outputs": [
```

Comment:
We don't need the `outputs` metadata. 
To remove them, once you run the notebook and all tests passed, restart the Kernel of the notebook, then copy-paste it into your test_notebook.ipynb